orm/models.py

```diff
@@ -396,14 +397,10 @@ def next_num(self):
         return self.page + 1
 
     async def next(self):
-        if self.has_next:
-            return await self.query.paginate(self.page + 1, self.per_page)
-        return await self.query.paginate(self.page, self.per_page)
+        return await self.query.paginate(self.page + 1, self.per_page)
```

Comment:
I have doubts about want would be the best to return when there are no more next pages.
Here would be return a empty when an user request next and there are no next page.
It's fine like this, or should we return the last page again? or raise a warning?